src/bloqade/builder/field.py

```diff
@@ -14,26 +14,92 @@ def uniform(self) -> "Uniform":
+            & F_{i}(t) = \sum_{\\alpha} C_{i}^{\\alpha}f_{\\alpha}(t) 
+            \\\\
+            & C_{i}^{\\alpha} \in \mathbb{R} 
+            \\\\
+            & f_{\\alpha}(t) \colon \mathbb{R} \\to \mathbb{R}
+            \\end{align*}
+            $$
+
+            The $i$-th component of the field is used to generate the drive at the $i$-th site.
+
+            Note that the drive is only applied if the $i$-th site is filled with an atom.
+
+            You build fields in Bloqade by first specifying the spatial modulation followed by the waveform
+            it should be multiplied by.
+
+            In the case of a uniform spatial modulation, it can be interpreted as 
+            a constant scaling factor where $C_{i}^{\\alpha} = 1.0$.
+
+        ??? example "Examples"
+
+            `uniform` can be accessed by first specifying the desired level coupling 
+            followed by the field to begin constructing.
+
+            ```python
+            from bloqade import start
+            geometry = start.add_position((0,0))
+            coupling = geometry.rydberg
+            uniform_rabi_amplitude = coupling.rabi.amplitude.uniform
+            ```
+
+        ??? info "Applications"
+
+            * [Single Qubit Rabi Oscillations](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-1-rabi/)
+            * [Single Qubit Ramsey Protocol](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-1-ramsey/)
+            * [Single Qubit Floquet Dynamics](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-1-floquet/)
+            * [Two Qubit Adiabatic Sweep](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-2-two-qubit-adiabatic/)
+            * [Multi-qubit Blockaded Rabi Oscillations](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-2-multi-qubit-blockaded/)
+            * [Nonequilibrium Dynamics of nearly Blockaded Rydberg Atoms](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-2-nonequilibrium-dynamics-blockade-radius/)
+            * [1D Z2 State Preparation](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-3-time-sweep/)
+            * [2D State Preparation](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-3-2d-ordered-state/)
+            * [Quantum Scar Dynamics](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-4-quantum-scar-dynamics/)
+            * [Solving the Maximal Independent Set Problem on defective King Graph](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-5-MIS-UDG/)
+            * [Lattice Gauge Theory Simulation](https://queracomputing.github.io/bloqade-python-examples/latest/examples/example-6-lattice-gauge-theory/)
+
+        ??? warning "Potential Pitfalls"
+
+            # Global vs. Local Detuning
```

Comment:
```suggestion
            # Global vs. Local Detuning on Hardware
```

Just make it clear from the title that this is hardware specific issue.